README.md

```diff
@@ -127,4 +127,4 @@ by doing the following:
 
 Set env variable IONOS_PINNED_CERT=<insert_sha256_public_fingerprint_here>
 
-You can get the sha256 fingerprint most easily form the browser by inspecting the certificate.
\ No newline at end of file
+You can get the sha256 fingerprint most easily form the browser by inspecting the certificate.
```

Comment:
I just now see that I wrote 'form' instead of 'from'. I know it's not from you, but can you change that too please?